LICENSE.md

```diff
@@ -1,33 +1,13 @@
-# Dual-License for W3C Test Suites
+# 3-Clause BSD License
```

Comment:
This PR seems nearly equivalent to #11191 except that this doesn't say "The" at the start and the "Redistribution ..." line appears modified in the diff, whereas it doesn't seem to be.

Reply:
I've compared it again and the only difference is indeed "The".

I also compared to the license file that GitHub gives you if you create a new repo with a license, and the only non-whitespace difference was that they call it "BSD 3-Clause License".

I've pushed @jugglinmike's changes to this PR and restored the documentation link and intend to merge this PR once checks have passed.